integrationTests/basic/jwt_auth.test.js

```diff
@@ -1,5 +1,6 @@
 jest.mock('@actions/core');
 jest.mock('@actions/core/lib/command');
+jest.setTimeout(30000);
```

Comment:
Is the timeout change still needed if the rsa keys are being replaced?

Reply:
Oh, nope, this isn't necessary anymore. Accidentally left in from testing.